### ISSUE TITLE
Connect Rocket.Chat with Nextcloud in order to build a strong communication platform

### ISSUE THREAD

**Issue by GoetheG** [nextcloud/server#9023]
Nextcloud as such offers collaboration features like circles, file sharing and so on. With Nextcloud Talk an app was introduced for self hosted video conferences. Due to the fact that Nextcloud is pushing towards a central communication device, wouldn't it be great to combine Nextcloud with Rocket.Chat? Maybe implement Rocket.Chat into Nextcloud and also Nextcloud Talk? Rocket.Chat is open source.

**Comment by MarPiRK** [comment]
In my company we're using Nextcloud with Rocket.Chat as external app, so it opens within the NextCloud's scope. We use LDAP as user backend, so there is no problem with sharing users, but we're thinking about giving OAuth2 a try (so that NextCloud is OAuth2 provider and Rocket.Chat is a client). That's some basic integration possible at the moment.
I've considered different approaches in Rocket.Chat/Rocket.Chat#7791 issue.

**Comment by GoetheG** [comment]
Hey @MarPiRK 
Very interesting what you wrote. Can you please tell me a little bit more about your connection between Rocket.Chat and Nextcloud?

**Comment by Aninstance** [comment]
I was just thinking along the same lines. There are a few FLOSS alternatives for Slack style collaboration/chat services (Rocket.Chat, Mattermost, Riot.im), and it seems to me like the seamless integration of one of these into NextCloud could represent another step towards positioning NC as a really competitive all-round productivity solution for SME, non-profits etc, education, etc.

Personally, I use both NC and Rocket.Chat and think they'd make a great pairing. But also would be happy to see open source alternatives that offers similar features if Rocket.Chat isn't deemed a good fit for NC, for whatever reason.

**Comment by isabellarussell** [comment]
Good news guys! It's happening! https://rocket.chat/2018/09/17/RC-Nextcloud-partnership-post/

**Comment by GoetheG** [comment]
Yes, @isabellarussell! It really happened! I am really happy that after I started this topic in March 2018 this kicked off.

**Comment by Elfanar** [comment]
No information how to authenticate nextcloud users on rocket chat seamlessly ?
